### PR TITLE
Parallelize bootstrap for latent position test

### PR DESCRIPTION
**Pull request by j1c**
#### What does this implement/fix? Briefly explain your changes.
- Added parallelized computation of bootstrap null

#### Any other comments?
None

### CODE REVIEW

File: graspologic/inference/latent_position_test.py [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
 
 def latent_position_test(
-    A1, A2, embedding="ase", n_components=None, test_case="rotation", n_bootstraps=500
+    A1,
+    A2,
+    embedding="ase",
+    n_components=None,
+    test_case="rotation",
+    n_bootstraps=500,
+    workers=1,
```
Comment (bdpedigo) [inline graspologic/inference/latent_position_test.py:25]: only comment is whether to call this workers or n_jobs like sklearn/joblib. i think somewhere else in inference we call it workers to match hyppo but in autogmm it's n_jobs

**Review by bdpedigo — APPROVED** [review]

File: graspologic/inference/latent_position_test.py (hunk 1 above)
Comment (j1c) [reply graspologic/inference/latent_position_test.py:25]: yea i was looking for `n_jobs`, but saw that LDT has `workers`. I do like n_jobs more than workers. I can change to `n_jobs` if we all agree

Comment (bdpedigo) [reply graspologic/inference/latent_position_test.py:25]: ya idc too much, would probably just be good to standardize across package i.e. the other stuff in `.inference`
@alyakin314 thoughts?

**Comment by bdpedigo** [comment]
@j1c any reason not to merge? would like to use this for something

File: graspologic/inference/latent_position_test.py (hunk 1 above)
Comment (bdpedigo) [reply graspologic/inference/latent_position_test.py:25]: i vote revisiting this in https://github.com/microsoft/graspologic/issues/755 and just sticking with workers for now for consistency's sake

Comment (j1c) [reply graspologic/inference/latent_position_test.py:25]: That sounds good

**Pull request merged**